NEWS.rst

```diff
@@ -34,13 +34,17 @@ Biopython 1.67).
 Sequences now have a ``defined`` attribute that returns a boolean indicating
 if the underlying data is defined or not.
 
+Add new ``gc_content`` function in ``SeqUtils`` and marks ``GC`` for future
+deprecation.
+
```

Comment:
Needs a short entry in ``DEPRECATED.rst`` too, under the existing ``SeqUtils`` section please.